src/main/resources/templates/search.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org/">
+<head th:replace="fragments :: head">
+</head>
+<body>
+
+<div class="container body-content">
+
+    <h1 th:text="${#strings.capitalizeWords(title)}"></h1>
+
+    <form th:action="@{/search/results}" method = "post">
+        <h2>Search by:</h2>
+
+
+        <p>
+            <span th:each="column : ${columns}">
+                <input
+                        type="radio"
+                        name="searchType"
+                        th:id="${column.key}"
+                        th:value="${column.key}"
+                        th:checked="${column.key == 'all'}"/>
+                <label th:for="${column.key}" th:text="${column.value}"></label>
+            </span>
+
+        </p>
+
+        <p>
+            <label for="searchTerm">KeyWord:</label>
+            <input type="text" name="searchTerm" id="searchTerm" />
+
+        </p>
+
+        <input type="submit" value="Search" />
+        <a href=" /adv-search">Advanced Search</a>
+
+    </form>
+
+    <hr />
+
+
+    <table class="recipe-listing">
+        <tr th:each="recipe : ${recipes}">
+            <td>
+                <p>
+                    Recipe Id:
+                    <a th:href="@{'/view/' + ${recipe.id}}" th:text="${recipe.id}"></a>
+                </p>
+                <p th:text="${'Name: ' + recipe.name}"></p>
+                <p>
+                    Name:
+                    <a th:href="@{'/names/view/' + ${recipe.name.id}}" th:text="${recipe.name}"></a>
+                </p>
+                <p>
+                    IngredientList:
+                <ul th:each="ingredientlist : ${recipe.ingredientLists}">
+                    <li>
+                        <a th:href="@{'/ingredientLists/view/' + ${ingredientList.id}}" th:text="${ingredientList.name}"></a>
+                    </li>
+                </ul>
+                </p>
+                <p>
+                    Tags:
+                <ul th:each="tags : ${recipe.tags}">
```

Comment:
this should be 

th:each="tag : ${recipe.tags}"

the code wont know which is the list and which is the individual list element, you'll have to name the iterable variable something unique and not the same as something else or else the computer wont know if youre referring to the list or the iterable variable

Reply:
If its written like this it should work now?
                    Tags:
                <ul>
                <li th:each text="${recipe.tags}"></li>
                    </p>
                </ul>

Reply:
Sean...Sean...SEAN!!!!!!! I have a list!!!!!  All day I still didn't have a list. It went to HomeController but not on my search!!! 
![image](https://github.com/CopeBrandon/recipe-project/assets/125934039/8b4c1190-862d-43b4-aac5-34b44b82bf4b)